std/socket.d

```diff
@@ -454,6 +450,7 @@ class Protocol
 version(CRuntime_Bionic) {} else
 @safe unittest
 {
+    // import std.stdio : writefln;
```

Comment:
writefln is used in commented out lines below. I put in this comment because before it was covered by the version(unittest) block.